### PR TITLE
[JDBC External Table] Phase 1: support create jdbc resource

### PR DESCRIPTION
**Pull request by silverbullet233**
## What type of PR is this：
- [ ] bug
- [x] feature
- [ ] enhancement
- [ ] others

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

support create jdbc resource

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/catalog/JDBCResource.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+* External JDBC resource for JDBC table query
+* <p>
+* Example:
+* CREATE EXTERNAL RESOURCE "jdbc_pg"
+* PROPERTIES
+* (
+* "type" = "jdbc",
+* "user" = "user",
+* "password" = "password",
+* "hosts" = "127.0.0.1:1234,127.0.0.2:1235",
```
Comment (mofeiatwork) [inline fe/fe-core/src/main/java/com/starrocks/catalog/JDBCResource.java:23]: what's the meaning of multiple hosts ?

File: fe/fe-core/src/test/java/com/starrocks/catalog/JDBCResourceTest.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+        Map<String, String> configs = new HashMap<>();
+        configs.put("hosts", "host1,host2");
+        configs.put("user", "user");
+        configs.put("password", "password");
+        configs.put("driver", "driver");
+        configs.put("jdbc_type", "jdbc_type");
+        configs.put("socketTimeout", "1000");
+        JDBCResource jdbcResourceWrite = new JDBCResource("jdbc_resource_test", configs);
+        jdbcResourceWrite.write(dos);
```
Comment (mofeiatwork) [inline fe/fe-core/src/test/java/com/starrocks/catalog/JDBCResourceTest.java:39]: Could we leverage some serialization testing framework, which could inject data into the serializable field automatic? Cause test serialization is boring

File: fe/fe-core/src/main/java/com/starrocks/catalog/JDBCResource.java (hunk 1 above)
Comment (silverbullet233) [reply fe/fe-core/src/main/java/com/starrocks/catalog/JDBCResource.java:23]: @mofeiatwork jdbc support multiple hosts in connection url (https://jdbc.postgresql.org/documentation/head/connect.html).

`hosts`will be deprecated and replaced by `jdbc_url`, I will update this PR later.

File: fe/fe-core/src/test/java/com/starrocks/catalog/JDBCResourceTest.java (hunk 2 above)
Comment (kangkaisen) [reply fe/fe-core/src/test/java/com/starrocks/catalog/JDBCResourceTest.java:38]: I think you could refer to HiveResourceTest.testSerialization

**Comment by silverbullet233** [comment]
run starrocks_fe_unittest

File: fe/fe-core/src/test/java/com/starrocks/catalog/JDBCResourceTest.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+public class JDBCResourceTest {
+    private Analyzer analyzer;
+
+    @Before
+    public void setUp() {
+        analyzer = AccessTestUtil.fetchAdminAnalyzer(true);
+    }
+
+    @Test
+    public void testSerialization() throws Exception {
```
Comment (kangkaisen) [inline fe/fe-core/src/test/java/com/starrocks/catalog/JDBCResourceTest.java:27]: should test the exception case. for example， the config key is 'xxxx'

Comment (silverbullet233) [reply fe/fe-core/src/test/java/com/starrocks/catalog/JDBCResourceTest.java]: @kangkaisen fixed

**Review by kangkaisen — APPROVED** [review]

**Review by Seaven — APPROVED** [review]

**Pull request merged**